keyboards/technika/technika.c

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 Álvaro "Gondolindrim" Volpato  <gondolindrim@acheronproject.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "technika.h"
+
+void keyboard_pre_init_kb(void) {
+    setPinOutput(A15);
+    setPinOutput(B3);
+    setPinOutput(B4);
+
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(A15, led_state.num_lock);
+        writePin(B3, led_state.caps_lock);
+        writePin(B4, led_state.scroll_lock);
+    }   
+    return true;
+}
```

Comment:
https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

Reply:
Specifically, if you have the defines, you don't need the code, at all.

Reply:
The idea here is to have the LEDs programmable for both CAPS, NUM, LOCK or as layer indicators. As I can see from the indicator documentation you linked, I don't see how those defines allow for the LEDs to be configured as layer indicators, or do they?

Reply:
You can override the default behaviour by adding the `led_update_user()` function in your keymap, returning false, and using the defines in `layer_state_set_user()` instead.

Reply:
Nice ok guys committed in 66faf5b22